diff --git a/docker.spec b/docker.spec
index 064d48d..fb519ba 100644
--- a/docker.spec
+++ b/docker.spec
@@ -18,7 +18,7 @@
 %global tar_import_path code.google.com/p/go/src/pkg/archive/tar
 
 # docker-selinux conditional
-%if 0%{?fedora} >= 22 || 0%{?centos} >= 7 || 0%{?rhel} >= 7
+%if 0%{?fedora} > 22 || 0%{?centos} >= 7 || 0%{?rhel} >= 7
 %global with_selinux 1
 %endif
 
@@ -43,13 +43,14 @@
 %global selinux_policyver 3.13.1-119
 %endif # with_selinux
 
+%global gopath /usr/share/gocode
+
 Name: %{repo}
 Version: 1.7.0
 Release: 5.git%{d_shortcommit}%{?dist}
 Summary: Automates deployment of containerized applications
 License: ASL 2.0
 URL: http://www.%{repo}.com
-ExclusiveArch: x86_64 %{arm}
 #Source0: https://%{import_path}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
 Source0: https://github.com/lsm5/%{repo}/archive/%{d_commit}/%{repo}-%{d_shortcommit}.tar.gz
 Source1: %{repo}.service
@@ -61,9 +62,11 @@ Source6: %{repo}-network.sysconfig
 %if 0%{?with_selinux}
 Source7: https://github.com/fedora-cloud/%{repo}-selinux/archive/%{ds_commit}/%{repo}-selinux-%{ds_shortcommit}.tar.gz
 %endif # with_selinux
+Patch0: docker-gccgo.patch
+Patch1: docker-gccgo-aarch.patch
 BuildRequires: git
 BuildRequires: glibc-static
-BuildRequires: golang >= 1.3.3
+BuildRequires: gcc-go >= 5
 BuildRequires: go-md2man
 BuildRequires: device-mapper-devel
 BuildRequires: btrfs-progs-devel
@@ -106,8 +109,8 @@ and tests on a laptop will run at scale, in production*, on VMs, bare-metal
 servers, OpenStack clusters, public instances, or combinations of the above.
 
 %package devel
-BuildRequires: golang >= 1.2.1-3
-Requires: golang >= 1.2.1-3
+BuildRequires: gcc-go >= 5
+Requires: libgo >= 5
 Provides: %{repo}-io-devel = %{version}-%{release}
 Provides: %{repo}-pkg-devel = %{version}-%{release}
 Provides: %{repo}-io-pkg-devel = %{version}-%{release}
@@ -262,8 +265,12 @@ This package installs %{summary}.
 
 %prep
 %autosetup -Sgit -n %{repo}-%{d_commit}
+
 cp %{SOURCE5} .
 
+# use defines from 32-bit ARM, should be the same on 64-bit ARM
+cp vendor/src/github.com/kr/pty/ztypes_arm.go vendor/src/github.com/kr/pty/ztypes_arm64.go
+
 %if 0%{?with_selinux}
 # unpack %{repo}-selinux
 tar zxf %{SOURCE7}
