--- golang-github-cpuguy83-go-md2man/golang-github-cpuguy83-go-md2man.spec	2015-04-17 09:38:42.554536847 +0200
+++ golang-github-cpuguy83-go-md2man.spec	2015-04-17 09:51:21.841067615 +0200
@@ -6,6 +6,7 @@
 %global import_path     %{provider}.%{provider_tld}/%{project}/%{repo}
 %global commit          2831f11f66ff4008f10e2cd7ed9a85e3d3fc2bed
 %global shortcommit     %(c=%{commit}; echo ${c:0:7})
+%global gopath /usr/share/gocode
 
 Name:           golang-%{provider}-%{project}-%{repo}
 Version:        1
@@ -14,7 +15,6 @@
 License:        MIT
 URL:            https://%{import_path}
 Source0:        https://%{import_path}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
-ExclusiveArch:  %{ix86} x86_64 %{arm}
 Provides:       %{repo} = %{version}-%{release}
 
 %description
@@ -22,9 +22,9 @@
 manpages.
 
 %package devel
-BuildRequires:  golang >= 1.2.1-3
+BuildRequires:  gcc-go
 BuildRequires:  golang(github.com/russross/blackfriday)
-Requires:       golang >= 1.2.1-3
+Requires:       gcc-go
 BuildArch:      noarch
 Summary:        A golang registry for global request variables
 Provides:       golang(%{import_path}) = %{version}-%{release}
@@ -46,7 +46,7 @@
 
 export GOPATH=$(pwd)/_build:%{gopath}
 pushd $(pwd)/_build/src
-go build  %{import_path}
+go build -compiler gccgo -gccgoflags "$RPM_OPT_FLAGS" %{import_path}
 popd
 
 %install
