--- golang-github-russross-blackfriday/golang-github-russross-blackfriday.spec	2015-04-17 09:40:27.691886578 +0200
+++ golang-github-russross-blackfriday.spec	2015-04-17 09:51:21.841067615 +0200
@@ -6,6 +6,7 @@
 %global import_path     %{provider}.%{provider_tld}/%{project}/%{repo}
 %global commit          77efab57b2f74dd3f9051c79752b2e8995c8b789
 %global shortcommit     %(c=%{commit}; echo ${c:0:7})
+%global gopath /usr/share/gocode
 
 Name:       golang-%{provider}-%{project}-%{repo}
 Version:    1.2
@@ -24,9 +25,9 @@
 %{summary}
 
 %package devel
-BuildRequires:  golang >= 1.2.1-3
+BuildRequires:  gcc-go
 BuildRequires:  golang(github.com/shurcooL/sanitized_anchor_name)
-Requires:   golang >= 1.2.1-3
+Requires:   gcc-go
 Requires:   golang(github.com/shurcooL/sanitized_anchor_name)
 Summary:    Markdown processor implemented in Go
 Provides:   golang(%{import_path}) = %{version}-%{release}
@@ -49,7 +50,7 @@
 cp -pav upskirtref/* %{buildroot}%{gopath}/src/%{import_path}/upskirtref/
 
 %check
-GOPATH=%{gopath}:%{buildroot}%{gopath} go test %{import_path}
+GOPATH=%{gopath}:%{buildroot}%{gopath} go test -compiler gccgo -gccgoflags "$RPM_OPT_FLAGS" %{import_path}
 
 %files devel
 %doc README.md
